File: ecommerc_dotnet/dto/Request/OrderRequestItemsDto.cs
namespace ecommerc_dotnet.dto.Request;

public class OrderRequestItemsDto
{
    public Guid storeId { get; set; }
    public decimal price { get; set; }
    public int quanity { get; set; } = 1;
    public Guid productId { get; set; }
    public List<Guid>  products_varientId { get; set; }
}

File: eccorce_app/app/src/main/java/com/example/eccomerce_app/dto/request/OrderRequestItemsDto.kt
package com.example.e_commercompose.dto.request

import com.example.hotel_mobile.services.kSerializeChanger.UUIDKserialize
import com.example.hotel_mobile.services.kSerializeChanger.UUIDListKserialize
import kotlinx.serialization.Serializable
import java.util.UUID

@Serializable
data class OrderRequestItemsDto(
    @Serializable(with = UUIDKserialize::class)
    val storeId: UUID,
    val price: Double,
    val quanity: Int,
    @Serializable(with = UUIDKserialize::class)
    val productId: UUID,
    @Serializable(with = UUIDListKserialize::class)
    val productsVarientId: List<UUID>,
)
